tests/specs/send/send-btc.spec.ts

```diff
@@ -36,5 +39,30 @@ test.describe('send btc', () => {
       const details = await sendPage.confirmationDetails.allInnerTexts();
       test.expect(details).toBeTruthy();
     });
+
+    // test('that asset value and recipient on preview match input', async ({ sendPage }) => {
```

Comment:
If you don't want a test to run, you can use `test.skip(...`

Reply:
Or, if you only want specific tests to run, you can flag them with `test.only(...`